Makefile

```diff
@@ -4,6 +4,6 @@
 REPO_ROOT := $(CURDIR)
 
 # Versions for tools that are not managed by devbox.
-ENVTEST_VERSION=1.29.x
+ENVTEST_VERSION=1.29.x!
```

Comment:
LGTM. this was adding `KUBEBUILDER_ASSETS` variable to the file. 
however I could not figure importance to `!` here. It didnt mattered if I use `1.29.x` or `1.29.x!`

Reply:
Ah sorry I'll add a comment, the ! means always use the latest available patch for this minor.